lib/ui/correct_news.dart

```diff
@@ -39,18 +47,90 @@ class _CorrectNewsScreenState extends State<CorrectNewsScreen> {
     }
   }
 
+  progressBarTimer() async {
+    timer = Timer.periodic(
+        const Duration(milliseconds: 30), (_) => setProgressBarValue());
+  }
+
+  setProgressBarValue() {
+    setState(() {
+      if (barValue < 1) {
+        barValue = barValue + 0.01;
+      } else {
+        timer?.cancel();
+      }
+    });
+  }
+
   @override
   Widget build(BuildContext context) {
+    var screenWidth = MediaQuery.of(context).size.width;
+    final appBarHeight = AppBar().preferredSize.height;
+
     return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          'Rodada ${gc.game.roundIndex + 1}/3',
+          style: const TextStyle(
+              fontSize: 16, fontWeight: FontWeight.bold, color: kYellowButton),
+        ),
+        automaticallyImplyLeading: false,
+      ),
       body: SafeArea(
         child: Column(
-          children: const [
-            Text('Link para a notícia real'),
-            Text('Noticia correta'),
-            Text('Lorem ipsum dolor sit amet, consectetur {RESPOSTA X} '
-                'elit ut aliquam, purus sit amet luctus venenatis, lectus'),
-          ],
-        ),
+            crossAxisAlignment: CrossAxisAlignment.stretch,
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Stack(
+                alignment: AlignmentDirectional.center,
+                children: [
+                  LinearProgressIndicator(
+                      minHeight: AppBar().preferredSize.height * 0.66,
+                      value: barValue,
+                      color: kPlayer_3),
+                  Text('Link para a notícia real'),
+                ],
+              ),
+              Container(
+                padding: EdgeInsets.only(
+                    left: screenWidth * 0.1,
+                    right: screenWidth * 0.1,
+                    bottom: appBarHeight * 0.5),
+                child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.end,
+                    children: [
+                      Container(
+                          padding: EdgeInsets.only(bottom: appBarHeight * 0.26),
+                          alignment: Alignment.center,
+                          child: const Text('Noticia correta',
+                              style: TextStyle(
+                                  fontSize: 20,
+                                  fontWeight: FontWeight.bold,
+                                  color: kYellowButton))),
+                      const ChatBalloon(
+                          color: Colors.white,
+                          balloonHeader: Text('Você ouviu que ...',
+                              style: TextStyle(
+                                  height: 1.5,
+                                  fontSize: 16,
+                                  fontWeight: FontWeight.bold,
+                                  color: kBackgroundDarkestGray)),
+                          balloonText: Text(
```

Comment:
#16 was merged and the ChatBalloon now takes a `RichText`as the `balloonText`.

Reply:
updated baloonText to RichText